src/Compatibility/Core/src/Android/Compatibility.Android.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
   <PropertyGroup>
     <Description>Android Compatibility Backend for Microsoft MAUI</Description>
-    <AssemblyName>Microsoft.Maui.Controls.Compatibility</AssemblyName>
+    <AssemblyName>Microsoft.Maui.Controls.Compatibility.Android</AssemblyName>
```

Comment:
MSBUILD can't handle this project and the iOS one having the same assembly name. It gets confused. Since we aren't packaging these versions of the dlls this is hopefully fine and doesn't ultimately lead to weird confusing scenarios